classes/jats/Repository.php

```diff
@@ -0,0 +1,221 @@
+<?php
+/**
+ * @file classes/jats/Repository.php
```

Comment:
I'm ambivalent about the use of a repository class for this. Actually, almost everything here is closely tied to the handling code, and will only be called once from the `PKPJatsController` -- so I'd be happy if it these functions were moved into `PKPJatsController` and this controller was removed. Then `classes/jats/JatsFile.php` could be moved into `lib/pkp/classes/submissionFile` alongside the parent class, which makes more sense to me.

But if you're not sure that's the best approach, I'd be OK to merge this as is too.

Reply:
I had the relationship with the `SubmissionFile` and the `SubmissionFile/Repository` dropped for `JatsFile` object, and that's why I took both files out of the SubmissionFile folder and in it's own `jats` folder. This can be used as the "code home" for existing business logic and future possible merging of the code from, let's say, `jatsTemplate` plugin for generating the "default" Jats content.